haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -201,6 +201,28 @@ managementSpec originate = do
 
         lExpectViewConsumerStorage consumer [balanceExpected]
 
+    it "removes zero balance accounts" $ integrationalTestExpectation $ do
```

Comment:
I put it in Management since despite transfer operation being specified in FA2, the specific behavior is not part FA2.

>AFAIU it is inside describe "Contract pausing". Why?

I think It was a mistake, will put it in separate section (or move to FA2 tests if it makes sense)

Reply:
Moved to a new "describe" section for transfer

Reply:
But "Management" in the module name means that this module tests management operations, i. e. those available only to managers (contract owner, master minter, pauser, etc.).
`Lorentz.Contracts.Test.FA2` tests this specific contract, not a general FA2 contract. For example, there is `validates token id` test which checks that `token_id = 1` is rejected. FA2 doesn't require `token_id = 1` to be rejected. And `OriginationParams` doesn't specify which token_ids are permitted. So I treat `Lorentz.Contracts.Test.FA2` as a module that tests entrypoints defined in FA2, keeping stablecoin specifics in mind.

Reply:
I see. I ll move it then.

Reply:
Moved.